Havit.Blazor.Components.Web.Bootstrap/Forms/Internal/HxInputDateInternal.razor.cs

```diff
@@ -63,6 +67,10 @@ public partial class HxInputDateInternal<TValue> : InputBase<TValue>, IAsyncDisp
 
 	[Parameter] public DateTime CalendarDisplayMonth { get; set; }
 
+	[Parameter] public string NameAttributeValue { get; set; }
+
+	[Parameter(CaptureUnmatchedValues = true)] public IReadOnlyDictionary<string, object> AdditionalAttributes { get; set; }
```

Comment:
I think the internal component should not use `CaptureUnmatchedValues = true` as the `HxInputDate` component should set the `AdditionalAttributes` parameter explicitly?

Reply:
I.e.
```diff csharp
-builder.AddMultipleAttributes(300, AdditionalAttributes);
+builder.AddAttribute(300, nameof(HxInputDateInternal<TValue>.AdditionalAttributes)), AdditionalAttributes);
```

Reply:
(I understand the original `AdditionalAttributes` came from `InputBase` with `CaptureUnmatchedValues`, but since this is an internal component, we can take advantage of the fact that it's only called from `HxInputDate`, where the `AdditionalParameters` parameter can be passed in its entirety.)